aptos-move/replay-benchmark/src/state_view.rs

```diff
@@ -35,7 +35,30 @@ pub(crate) struct ReadSetCapturingStateView<'s, S> {
 }
 
 impl<'s, S: StateView> ReadSetCapturingStateView<'s, S> {
-    pub(crate) fn new(state_view: &'s S, initial_read_set: HashMap<StateKey, StateValue>) -> Self {
+    pub(crate) fn new(
+        state_view: &'s S,
+        mut initial_read_set: HashMap<StateKey, StateValue>,
+    ) -> Self {
+        // For generating and capturing reads, benchmark uses sequential execution. This means that
```

Comment:
it says uses sequential execution - but then about speculation.. Does it mean to say parallel execution?

Reply:
Maybe my writing is bad 😄

What I meant to say: 
1. We use sequential execution, so read-set captures the true values only.
2. We use parallel execution for benchmark, so speculatively execution can read something off the successful path and use a read not captured by (1)
3. Currently I panic in benchmark if there is a read outside to be safe to detect errors like sending too many requests to API, etc.
4. Was observing panics because `0x1::error` module not found because the sequential execution never called that, but parallel did.

Reply:
oh, got it! thanks makes sense